notebooks/run_model_on_snellius.ipynb

```diff
@@ -0,0 +1,300 @@
+{
```

Comment:
<p>Please note that the config_file_snellius.txt is a template. So it is better to not modify it. Perhaps we can just ask the user to give us the path to a config file e.g. <code>config_path = ./config.txt.</code>Then here it is loaded as a dictionary. So, it is editable. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/EcoExtreML/processing/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>